Infusionsoft/InvoiceServiceBase.php

```diff
@@ -138,10 +140,7 @@ public static function createBlankOrder($contactId, $description, $orderDate, $l
         );
 
         $result = parent::send($app, "InvoiceService.createBlankOrder", $params);
-        $invoice = new Infusionsoft_Invoice($result);
-        $job = new Infusionsoft_Job($invoice->JobId);
-        Infusionsoft_SdkEventManager::dispatch(new Infusionsoft_SdkEvent($invoice, array('result' => $result)), 'DataObject.Saved');
-        Infusionsoft_SdkEventManager::dispatch(new Infusionsoft_SdkEvent($job, array('result' => $result)), 'DataObject.Saved');
+        Infusionsoft_SdkEventManager::dispatch(new Infusionsoft_SdkEvent($result, array('result' => $result)), 'InvoiceService.OrderCreated');

```

Comment:
There's no reason to send $result twice, the array can be left out and just the invoiceId (which is $result) be sent